plugins/admin_console/system_partner/lib/api/KalturaSystemPartnerConfiguration.php

```diff
@@ -871,6 +871,11 @@ public function toObject ( $object_to_fill = null , $props_to_skip = array() )
 			$object_to_fill->setAllowedDomains('');
 		}
 		
+		if (is_null($this->cdnHostWhiteList))
+		{
+			$object_to_fill->setCdnHostWhiteList(null);
+		}
+		
```

Comment:
@yossipapi this is more of a 'quality of life' thing, and I think it's required
If we omit this part, it means in order to clear the cdnHostWhiteList value using admin console partner config we need to pass ' '
Because if we just clear the checkbox "CDN Host white list regex (comma seperated)" from partner config, toObject will skip it since it's null and will not modify the value
To bypass it, we can pass ' ' - which 'setCdnHostWhiteList' will now correctly identify it as empty and set null

However, I think it is confusing and incorrect way of saying "I want to remove this for the partner"
Therefore, now if that field is_null - we will set it.